AppCenterData/AppCenterData/Internal/Client/MSDataOperationProxy.m

```diff
@@ -69,14 +69,20 @@ - (void)performOperation:(NSString *_Nullable)operation
     if ([self shouldAttemptRemoteOperationForDocument:cachedDocument]) {
       MSLogInfo([MSData logTag], @"Performing remote operation");
       remoteDocumentBlock(^(MSDocumentWrapper *_Nonnull remoteDocument) {
+
         // If a valid remote document was retrieved, update local store
         if (remoteDocument.error == nil) {
+          
+          // For delete operation, delete the document from local cache.
+          // Otherewise, for other online scenarios, the intended pending operation in the local store is nil.
```

Comment:
"Otherewise" -> "Otherwise", also not a blocker but we usually use '/*...*/' for multiline comments on iOS.